app/controllers/talks_controller.rb

```diff
@@ -1,4 +1,6 @@
 class TalksController < ApplicationController
+  before_action :require_logged_in
```

Comment:
I want to allow accsess to talks page without login.

Reply:
Thank you for reviewig. Fixed.